### PR TITLE
Adding new sample scripts which leverage Apps and Management SDKs

### PR DESCRIPTION
**Pull request by hemanshu-cohesity**
These are updated Cohesity Management SDK scripts, which leverages authentication using Cohesity Apps APIs. EasyScript App will have a reference to this samples folder in the App.

### CODE REVIEW

File: cohesity_app_sdk/samples/clone_and_update_view/clone_and_update_view.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        updated_view = json.loads(jsonpickle.encode(resp))
+
+        # Verify the fields updated
+        assert updated_view['protocol_access']== protocol_access,\
+            "View not updated"
+        assert updated_view['description'] == description, "View not updated"
+        print ("Updated view:")
+        pprint.pprint(updated_view)
+
+def get_mgnt_token():
```
Comment (pyashish) [inline cohesity_app_sdk/samples/clone_and_update_view/clone_and_update_view.py:72]: typo: get_mgmt_token()

File: cohesity_app_sdk/samples/clone_and_update_view/clone_and_update_view.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    mgmt_auth_token = token.create_management_access_token()
+    return mgmt_auth_token
+
+def main(args):
+
+    # Login to the cluster
+    host_ip = os.getenv('HOST_IP')
+    mgmt_auth_token = get_mgnt_token()
+    cohesity_client = CohesityClient(cluster_vip=host_ip,
+                                        auth_token=mgmt_auth_token)
```
Comment (pyashish) [inline cohesity_app_sdk/samples/clone_and_update_view/clone_and_update_view.py:104]: correct Intend

File: cohesity_app_sdk/samples/clone_and_update_view/clone_and_update_view.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        # Verify the fields updated
+        assert updated_view['protocol_access']== protocol_access,\
+            "View not updated"
+        assert updated_view['description'] == description, "View not updated"
+        print ("Updated view:")
+        pprint.pprint(updated_view)
+
+def get_mgnt_token():
+    """
+    To get the management access token from athena to authenticate
```
Comment (pyashish) [inline cohesity_app_sdk/samples/clone_and_update_view/clone_and_update_view.py:74]: Change Athena to App Server.

File: cohesity_app_sdk/samples/clone_and_update_view/clone_and_update_view.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    # Clone a view with name.
+    try:
+        cloneview = CloneView(cohesity_client)
+        cloneview.clone_existing_view(view_name, clone_name)
+
+        # Update the cloned view.
+        cloneview.update_view(view_name=clone_name,
+                              protocol_access=ProtocolAccessEnum.KNFSONLY,
+                              description="View to restrict access"
+                                          " to s3 only.")
```
Comment (pyashish) [inline cohesity_app_sdk/samples/clone_and_update_view/clone_and_update_view.py:123]: Isn't this KNFSONLY ?

File: cohesity_app_sdk/samples/list_protection_jobs/list_protection_jobs.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        """
+        Method to convert epoch time in usec to date format
+        :param epoch(int): Epoch time of the job run.
+        :return: date(str): Date format of the job runj.
+        """
+        date = datetime.datetime.fromtimestamp(epoch/10**6).\
+            strftime('%m-%d-%Y %H:%M:%S')
+        return date
+
+def get_mgnt_token():
```
Comment (pyashish) [inline cohesity_app_sdk/samples/list_protection_jobs/list_protection_jobs.py:38]: typo

File: cohesity_app_sdk/samples/list_protection_jobs/list_protection_jobs.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+    return mgmt_auth_token
+
+
+def main():
+
+    # Login to the cluster
+    host_ip = os.getenv('HOST_IP')
+    mgmt_auth_token = get_mgnt_token()
+    cohesity_client = CohesityClient(cluster_vip=host_ip,
+                                      auth_token=mgmt_auth_token)
```
Comment (pyashish) [inline cohesity_app_sdk/samples/list_protection_jobs/list_protection_jobs.py:70]: correct intend

File: cohesity_app_sdk/samples/list_unresolved_alerts/list_unresolved_alerts.py [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    mgmt_auth_token = token.create_management_access_token()
+    return mgmt_auth_token
+
+def main(args):
+
+    # Login to the cluster
+    host_ip = os.getenv('HOST_IP')
+    mgmt_auth_token = get_mgnt_token()
+    cohesity_client = CohesityClient(cluster_vip=host_ip,
+                                         auth_token=mgmt_auth_token)
```
Comment (pyashish) [inline cohesity_app_sdk/samples/list_unresolved_alerts/list_unresolved_alerts.py:77]: intend

File: cohesity_app_sdk/samples/on_demand_job_run/on_demand_job_run.py [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+                                                            num_runs=1)[0]
+        else:
+            jresp = jresp[0]
+        if jresp.backup_run.status == RunStatusEnum.KSUCCESS:
+            print ("Protection Job %s started successfully" % job_name)
+        elif jresp.backup_run.status == RunStatusEnum.KERROR:
+            print ("Protection Job %s failed." % job_name)
+
+
+def get_mgnt_token():
```
Comment (pyashish) [inline cohesity_app_sdk/samples/on_demand_job_run/on_demand_job_run.py:73]: typo

File: cohesity_app_sdk/samples/on_demand_job_run/on_demand_job_run.py [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+
+    # Get the management access token.
+    token = app_cli.token_management
+    mgmt_auth_token = token.create_management_access_token()
+    return mgmt_auth_token
+
+def main(args):
+    # Login to the cluster
+    host_ip = os.getenv('HOST_IP')
+    mgmt_auth_token = get_mgnt_token()
```
Comment (pyashish) [inline cohesity_app_sdk/samples/on_demand_job_run/on_demand_job_run.py:101]: typo

File: cohesity_app_sdk/samples/on_demand_job_run/on_demand_job_run.py [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+    cohesity_client = CohesityClient(cluster_vip=host_ip,
+                                         auth_token=mgmt_auth_token)
```
Comment (pyashish) [inline cohesity_app_sdk/samples/on_demand_job_run/on_demand_job_run.py:103]: Intend

**Review by pyashish — CHANGES_REQUESTED** [review]
LGTM most of the part, Please fix the comments. 
Also, just my 2c : I don't see how the samples are related to App SDK. Why would anyone just list from App sdk

**Comment by hemanshu-cohesity** [comment]
As discussed, moving these samples to another repo

**Pull request closed without merging**